test_suite/generic/test_generic.py

```diff
@@ -254,6 +254,24 @@ def test_grub_config(self, host):
 
             assert host.file(grub2_file).linked_to == linked_to
 
+    @pytest.mark.run_on(['rhel10.0'])
```

Comment:
```suggestion
    @pytest.mark.run_on(['rhel10'])
```
If we specify only major version, it will apply to all RHEL-10.x minor versions too (which is our intention unless something changes in later minor releases, then we update the test case or marker accordingly).